src/paginationCalls/resourceCollector.ts

```diff
@@ -14,10 +14,11 @@
  * limitations under the License.
  */
 
-import {
+ import {
```

Comment:
wired. we should not change here too.
You could try `npm run fix` before commit the code